src/property-filter/controller.ts

```diff
@@ -324,16 +325,23 @@ export const getAutosuggestOptions = (
   }
 };
 
-export const operatorToDescription = (operator: ComparisonOperator, i18nStrings: I18nStrings) => {
-  const mapping: { [K in ComparisonOperator]: string | undefined } = {
-    ['<']: i18nStrings.operatorLessText,
-    ['<=']: i18nStrings.operatorLessOrEqualText,
-    ['>']: i18nStrings.operatorGreaterText,
-    ['>=']: i18nStrings.operatorGreaterOrEqualText,
-    [':']: i18nStrings.operatorContainsText,
-    ['!:']: i18nStrings.operatorDoesNotContainText,
-    ['=']: i18nStrings.operatorEqualsText,
-    ['!=']: i18nStrings.operatorDoesNotEqualText,
-  };
-  return mapping[operator];
+export const operatorToDescription = (operator: SupportedOperator, i18nStrings: I18nStrings) => {
```

Comment:
This function is called from elsewhere passing an operator of type `PropertyFilterOperator`. Will these calls not break once collection-hooks is updated and therefore `PropertyFilterOperator` and `SupportedOperator` differ, with the former having more possible values?

Same with `getOperatorI18nString` further down.

Reply:
Right! Let me get back the defaults for that case.